tests/e2e/cucumber/environment/index.ts

```diff
@@ -13,34 +13,11 @@ import { Browser, chromium, firefox, webkit } from '@playwright/test'
 import path from 'path'
 import fs from 'fs'
 
-import { config } from '../../config'
-import { api, environment } from '../../support'
 import { World } from './world'
 import { state } from './shared'
-import {
-  createdSpaceStore,
-  createdLinkStore,
-  createdGroupStore,
-  createdUserStore,
-  keycloakCreatedUser,
-  federatedUserStore,
-  dummyKeycloakGroupStore
-} from '../../support/store'
+import { config } from '../../config'
 import { Group, User } from '../../support/types'
-import {
-  createdTokenStore,
-  federatedTokenStore,
-  keycloakTokenStore
-} from '../../support/store/token'
-import { removeTempUploadDirectory } from '../../support/utils/runtimeFs'
-import {
-  setAccessTokenForKeycloakUser,
-  setupKeycloakAdminUser,
-  refreshAccessTokenForKeycloakUser,
-  refreshAccessTokenForKeycloakOcisUser,
-  setAccessTokenForKeycloakOcisUser
-} from '../../support/api/keycloak'
-import { closeSSEConnections } from '../../support/environment/sse'
+import { api, environment, utils, store } from '../../support'
```

Comment:
I think the imports are organized accroding to the directory level. If so then it should be like this IMHO: just re-arranging their order
```ts
import { api, environment, utils, store } from '../../support'
import { Group, User } from '../../support/types'
```

Reply:
not sure about that, so need clarification on that. What i know is putting libraries import at top and internal import below that

Reply:
it's for organizing imports. good to have ordered imports (but no pressure)